assets/css/main.css

```diff
@@ -91,6 +91,33 @@ a:hover {
 
 /* FAQ
 ================================ */
+.accordion-button::after {
+  background-image: url(../img/plus-drop-down.svg);
+  }
+
+.accordion-button:not(.collapsed)::after {
+  background-image: url(../img/minus-drop-down.svg);
+}
+
+.accordion-button:focus {
+  border-color: rgba(0,0,0,.125);
+  box-shadow: none;
```

Comment:
Although it makes the page look better, I would recommend keeping the `box-shadow` for accessibility reasons. That way, users can tab through the FAQ sections and see where they are. In an ideal world, we'd use the [`:focus-visible`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible) selector, but it doesn't have great browser support.
```suggestion
```